src/components/menus/common-contextual-menu.tsx

```diff
@@ -60,22 +64,40 @@ export default function CommonContextualMenu({ menuItems, ...menuProps }: Readon
         );
     }
 
-    let dividerCount = 0;
-    return (
-        <StyledMenu keepMounted {...menuProps}>
-            {menuItems?.map((menuItem, index) => {
+    const renderMenuItems = useCallback(
+        (nodeMenuItems: MenuItemType[] | undefined) => {
+            let dividerCount = 0;
+            return nodeMenuItems?.map((menuItem, index) => {
                 if (menuItem.isDivider) {
                     dividerCount += 1;
                     return <Divider key={`divider${dividerCount}`} />;
                 }
-                return makeMenuItem(
-                    index,
-                    menuItem.messageDescriptorId,
-                    menuItem.callback,
-                    menuItem.icon,
-                    menuItem.disabled
+                if (menuItem.subMenuItems === undefined) {
+                    return makeMenuItem(
+                        index,
+                        menuItem.messageDescriptorId,
+                        menuItem.callback,
+                        menuItem.icon,
+                        menuItem.disabled
+                    );
```

Comment:
are you sure you dont want to use CustomMenuItem as in study to make the same hightlight effect?